cdk-infra/static/custom-environment-variables.json

```diff
@@ -21,6 +21,8 @@
   "fastlyCloudManagerServiceId": "FASTLY_CLOUD_MANAGER_SERVICE_ID",
   "gatsbyBaseUrl": "GATSBY_BASE_URL",
   "previewBuildEnabled": "PREVIEW_BUILD_ENABLED",
+  "featureFlagUpdatePages": "FEATURE_FLAG_UPDATE_PAGES",
```

Comment:
For my own edification -- why do we have this file and also the one at cdk-infa/static/config/custom-environment-variables.json ?

Reply:
Good question, we don't actually need this one I don't believe now that I think about it. We do need the `cd-infra/static/api/config`, though. This is because of how the Lambdas are bundled vs using Docker containers for the workers. When bundling the lambdas, the bundler can't access files above the root directory of the `cdk-infra` repo i.e. we can't get the these config JSON files from the root directory of `docs-worker-pool` so we have to copy them over.

 I'll remove it and test it to confirm that it still works.